app/Resources/views/subscription/tabs/tab-general.html.twig

```diff
@@ -46,7 +46,12 @@
 
     <div class="row">
         <div class="col-xs-11">
-            {{ form_row(form.contact.email) }}
+            <div class="form-group">
+                <label class="col-sm-2 control-label">{{ 'email'|humanize|trans }}</label>
```

Comment:
heh, [humanize](http://symfony.com/doc/current/reference/twig_reference.html#humanize), cool. But in this case what does it do? Just capitalize 'email' before translating it?